### PR TITLE
Update FirebaseRemoteConfig.java

### PR DESCRIPTION
**Pull request by teknepal**

### CODE REVIEW

**Comment by gugahoi** [comment]
Thanks for this change. Just so I am clear: is the current implementation incorrect? Does that method not exist?

**Comment by teknepal** [comment]
@gugahoi yes, current implementation for android is incorrect based on this: 
https://firebase.google.com/docs/remote-config/get-started?platform=android or           https://github.com/firebase/snippets-android/blob/77870d2c0f6654b5a7582d2d647495c49b04afd5/config/app/src/main/java/com/google/firebase/quickstart/config/java/MainActivity.java#L32-L35

**Comment by muhrifqii** [comment]
will this be merged? @gugahoi

**Comment by anonimitoraf** [comment]
@gugahoi looks like the method exists

image

However, I don't think that it's the right method to use. We want to use `setMinimumFetchIntervalInSeconds` (as @teknepal has in their PR)

image

**Review by anonimitoraf — COMMENTED** [review]
Thanks for the PR @teknepal ! Sorry for the delay, do you mind quickly resolving the merge conflict?

File: android/src/main/java/com/getcapacitor/community/firebaserc/FirebaseRemoteConfig.java [hunk 1]
```diff
@@ -44,9 +44,9 @@ public void load() {
 
   @PluginMethod
   public void initialize(PluginCall call) {
-    int minFetchTimeInSecs = call.getInt("minimumFetchIntervalInSeconds", 3600);
+    int minFetchTimeInSecs = call.getInt("minimumFetchInterval", 3600);
```
Comment (anonimitoraf) [inline android/src/main/java/com/getcapacitor/community/firebaserc/FirebaseRemoteConfig.java:47]: Good catch

**Comment by teknepal** [comment]
@anonimitoraf done, please merge and publish to npm.

**Review by gugahoi — APPROVED** [review]
Thank you for catching this @teknepal!

**Comment by gugahoi** [comment]
This is now in the 1.0.1 release

**Pull request merged**